lib/backend/k8s/k8s.go

```diff
@@ -325,8 +251,8 @@ func (c *KubeClient) Syncer(callbacks api.SyncerCallbacks) api.Syncer {
 func (c *KubeClient) Create(d *model.KVPair) (*model.KVPair, error) {
 	log.Debugf("Performing 'Create' for %+v", d)
 	switch d.Key.(type) {
-	case model.GlobalConfigKey:
-		return c.globalConfigClient.Create(d)
+	case model.GlobalFelixConfigKey:
```

Comment:
oops, I don't think we want to rename the `model.GlobalConfig` stuff.  That's a breaking change that requires corresponding Felix changes.